docusaurus/docs/reactnative/common-content/core-components/channel/props/input.mdx

```diff
@@ -1,5 +1,5 @@
 Component to render UI part of enclosed MessageInput component.
-Please read section in guides about [Custimizing Message Input](/reactnative/guides/message-input).
+Please read section in guides about [Custimizing Message Input](/reactnative/guides/message-input-customization).
```

Comment:
```suggestion
Please read section in guides about [Customizing Message Input](/reactnative/guides/message-input-customization).
```

Reply:
Pff, just spicing up the language :D (Thanks, nice catch!)